ocf/serviced-storage

```diff
@@ -121,6 +132,18 @@ END
 }
 
 ##############################################################################
+# Get the value of a serviced configuratoin property. Return empty string if
+# the property is not found.
+get_serviced_property() {
```

Comment:
Good idea.  I created this ticket https://jira.zenoss.com/browse/CC-1799 to modify serviced to print the values.  After CC-1799 is implemented, I'll update this PR